mlflow/store/model_registry/rest_store.py

```diff
@@ -197,8 +197,15 @@ def delete_registered_model_tag(self, name, key):
     # CRUD API for ModelVersion objects
 
     def create_model_version(
-        self, name, source, run_id=None, tags=None, run_link=None, description=None
-    ):
+        self,
+        name,
+        source,
+        run_id=None,
+        tags=None,
+        run_link=None,
+        description=None,
+        local_model_path=None,
+    ):  # pylint: disable=unused-argument
```

Comment:
```suggestion
    ):
```

Reply:
When overriding an abstract method, we don't need `# pylint: disable=unused-argument`.